### PR TITLE
Status pages and results UI

### PR DESCRIPTION
**Pull request by justinmc**
This updates the status pages with Malte's UI, and it modifies the run UI to use real data from the VDE run.

### CODE REVIEW

**Comment by justinmc** [comment]
@dionjwa You can review/take a look at this before I merge it.  It's mostly copying in Malte's UI stuff, but I also did a little bit of data reorganization with workflows and views on the frontend.  I also fixed a small 404 bug in the server.

File: client/public/js/records/workflow_record.js [hunk 1]
```diff
@@ -1,30 +1,18 @@
...
-  runId: null,
+  run: new RunRecord(),
   runs: 0,
-  status: statusConstants.IDLE,
   title: '',
-  uploadError: null,
-  uploadPending: false,
   views: 0,
   workflowNodes: new IList(),
 });
```
Comment (dionjwa) [inline client/public/js/records/workflow_record.js]: Where is this record saved? Is this going in the cache?

**Comment by dionjwa** [comment]
Justin, you are going to need to walk me through this application for me to do meaningful code reviews.

File: client/public/js/records/workflow_record.js (hunk 1 above)
Comment (justinmc) [inline client/public/js/records/workflow_record.js]: Sorry I forgot to reply to this.  Let me know if this still isn't clear after our talk earlier.  Basically these records are saved in stores on the client only, but they include a bunch of data that will usually also need to be saved in the backend (in Redis for now).

**Pull request merged**